### PR TITLE
fix: cve fix for kafka and jmx-javaagent

### PR DESCRIPTION
**Pull request by SandhyaRavi2403**
CVE fix for Kafka and jmx-javaagent

https://jira.nutanix.com/browse/NCN-102450
https://jira.nutanix.com/browse/NCN-102451

workflow:
https://github.com/mesosphere/dkp-container-images/actions/runs/10844306857

### CODE REVIEW

**Review by mhrabovcin — CHANGES_REQUESTED** [review]
@SandhyaRavi2403 we need keep the kafka `0.25.1` version in the `/services` directory and add the `0.25.2` with image bumps and changes.

**Comment by SandhyaRavi2403** [comment]
> @SandhyaRavi2403 we need keep the kafka `0.25.1` version in the `/services` directory and add the `0.25.2` with image bumps and changes.

@mhrabovcin the version is kept as is       version: 0.25.1, wherein the bump is updated. Do you mean not bump, get back to previous one `/services/0.25.1`

**Comment by mhrabovcin** [comment]
@SandhyaRavi2403 we will end up with 2 directories in the services dir. 0.25.1 and 0.25.2. We need to keep 0.25.1 as it is and add 0.25.2 with image bumps.

**Review by msdolbey — APPROVED** [review]
👍 
it is though

File: services/kafka-operator/0.25.2/kafka-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```
Comment (mhrabovcin) [inline services/kafka-operator/0.25.2/kafka-operator.yaml:2]:
```suggestion
apiVersion: helm.toolkit.fluxcd.io/v2beta2
```

**Comment by mhrabovcin** [comment]
@SandhyaRavi2403 the last commit title needs to be fixed here.

**Review by mhrabovcin — APPROVED** [review]

**Pull request merged**